install_config/aggregate_logging.adoc

```diff
@@ -163,6 +168,10 @@ To use a non-default project, create the project as usual using `oc new-project`
 |Specify the name of an existing pull
 secret to be used for pulling component images from an authenticated registry.
 
+|`openshift_logging_curator_image`
+|The image version for Curator.
+For example: `openshift_logging_curator_image=<registry_url>:5000/openshift3/logging-curator:v3.x.x`
```

Comment:
Shall we use offical images as an example?   registry.redhat.io/openshift3/ose-logging-curator5:v3.11. and it is curator5 in 3.11